rfcs/20200717-onednn-v2/api-simplified/v2_dnnl_threadpool.hpp

```diff
@@ -0,0 +1,27 @@
+#include "dnnl.hpp"
+#include "dnnl_threadpool_iface.hpp"
+
+namespace dnnl {
+namespace threadpool {
+
+// option 1a (preferable, but breaks backwards compatibility)
```

Comment:
Just wanted to note, that, as far as I understand, breaking the backwards compatibility here is not an issue. Changes for framework integration may be required for handling threadpool in oneDNN, regardless of potentially dropping stream attributes.

Reply:
That's true, however if someone used `stream_attr` (empty one) this code will stop working. So formally we break backwards compatibility even for non-threadpool case. I don't expect anyone does this though... (resolved)